recipes/aws-iam-authenticator/build.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+chmod a+x aws-iam-authenticator
+mv aws-iam-authenticator $PREFIX/bin
```

Comment:
Is this package just one script? Or are you moving a binary directly?

Reply:
I'm moving a binary directly.

I've tried to simply use the `script` clause in the `meta.yaml` but I was not successful.

Reply:
Did you try:
```
script:
  - chmod a+x aws-iam-authenticator  # [unix]
  - mv aws-iam-authenticator $PREFIX/bin  # [unix]
  - xcopy aws-iam-authenticator.exe %LIBRARY_BIN%  # [win]
```

I don't mind to have two files to execute it. It is not necessary to change

Reply:
That would probably work.

What would you prefer?

Reply:
You can let this as it is, using two separated files. That is fine :)